Tests/RxCocoaTests/NSComboBox+RxTests.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  NSComboBox+RxTests.swift
+//  Tests
+//
+//  Created by Jacob Gorban on 07/17/17.
+//  Copyright © 2017 Krunoslav Zaher. All rights reserved.
+//
+
+import RxSwift
+import RxCocoa
+import AppKit
+import XCTest
+
+final class NSComboBoxTests: RxTest {
+
+}
+
+extension NSComboBoxTests {
+    func testComboBox_IndexOfSelectedItemCompletesOnDealloc() {
+        let createView: () -> NSComboBox = { NSComboBox(frame: CGRect(x: 0, y: 0, width: 1, height: 1)) }
+        ensurePropertyDeallocated(createView, -1) { (view: NSComboBox) in view.rx.indexOfSelectedItem }
+    }
+
+    func testComboBox_ComboBoxSelectionDidChange_ForwardsToDelegates() {
+
+        var completed = false
+
+        autoreleasepool {
+            let comboBox = NSComboBox()
+            let delegate = ComboBoxDelegate()
+            comboBox.delegate = delegate
+            var rxDidChange = false
+
+            _ = comboBox.rx.indexOfSelectedItem
+                .skip(1) // Initial value
```

Comment:
@kzaher I was basing this test on your NSTextField+RxTests.swift file, which doesn't test for initial value. It's also pretty meaningless. The test is named testComboBox_ComboBoxSelectionDidChange_ForwardsToDelegates and doesn't actually check the values of the selected items (nor are they even set in the test).